README.md

```diff
@@ -152,25 +144,29 @@ a) z głosem stanowiącym – członkowie zwyczajni,
 
 b) z głosem doradczym – członkowie wspierający oraz zaproszeni goście.
 
-Walne Zebranie Członków może być zwyczajne i nadzwyczajne.
+Walne Zebranie Członków jest zwoływane w trybie zwyczajnym lub nadzwyczajnym. Zebranie odbywa się w siedzibie Stowarzyszenia lub zostaje przeprowadzone z wykorzystaniem środków komunikacji elektronicznej. Zarząd powiadamia członków Stowarzyszenia o terminie zebrania, planowanym porządku obrad oraz:
 
-Dla zwołania Walnego Zebrania Członków podaje się dwa terminy. Jeśli w pierwszym terminie nie zbierze się kworum, Walne Zebranie Członków odbywa się w drugim terminie. Członków Stowarzyszenia zawiadamia się o terminach Walnego Zebrania za pośrednictwem poczty elektronicznej.
+a) jego szczegółowej lokalizacji – w przypadku zebrania przeprowadzanego w siedzibie Stowarzyszenia,
 
-Aby Walne Zebranie Członków mogło się rozpocząć w pierwszym terminie wymagane jest kworum, czyli obecność co najmniej połowy ogólnej liczby zwyczajnych członków Stowarzyszenia. W drugim terminie kworum nie jest wymagane.
+b) sposobie dołączenia do spotkania – w przypadku zebrania przeprowadzanego z wykorzystaniem środków komunikacji elektronicznej
 
-Oba terminy Walnego Zebrania Członków nie mogą być odległe bardziej niż 14 dni kalendarzowych.
+za pośrednictwem poczty elektronicznej, w terminie uzależnionym od trybu organizacji zebrania.
```

Comment:
@kpc21 nie jest to show stopper, ale ja bym dał tę część jeszcze przed wypunktowaniem a) oraz b), dla czytelności.

Reply:
Naniesione. Choć szczerze, dla mnie czytelniej (przejrzyściej, nie nabźdźgane tak w jednym akapicie) było tak, jak ja zrobiłem, ale ty spojrzałeś na to bardziej z perspektywy kogoś z zewnątrz, kto to czyta, więc chyba możesz mieć więcej racji.